core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -43,28 +57,46 @@
         </ul>
       </div>
     </div>
+    <div class="oppia-collapse-content"
+         *ngIf="isContentOverflowing"
+         (click)="toggleExpansionState(0)">
+      {{ isContentExpanded ? "View Less" : "View All" }}
```

Comment:
Please make variables for these strings and use them here.

Reply:
Done